### ISSUE TITLE
Ally on API list page

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#11433]
### Describe your problem(s)

1.  top nav links has two focus 
2. Settings - ARIA attributes must conform to valid values
3. Admin - ARIA attributes must conform to valid values
4. Search button - Buttons must have discernible text
5. Table & Grid view buttons -  Buttons must have discernible text
6. h6 tag used for version - Heading levels should only increase by one
7. Missing H1 for page - Page should contain a level-one heading
8. Two focus for API card with image without test - Links must have discernible text
9. Top nav ul used -  `<ul> and <ol> must only directly contain <li>, <script> or <template> elements `
10. API icon and delete icon - id attribute value must be unique 7 of 7

![image](https://user-images.githubusercontent.com/20179540/125604858-63914895-bbb3-405a-bab9-857e492535ef.png)

### How will you implement it

---
